### PR TITLE
Scoring tests in Virtual Screening pipelines

### PR DESCRIPTION
**Pull request by mwojcikowski**

### CODE REVIEW

File: oddt/virtualscreening.py [hunk 1]
```diff
@@ -268,11 +270,17 @@ def score(self, function, protein=None, *args, **kwargs):
             extension = protein.split('.')[-1]
             protein = six.next(toolkit.readfile(extension, protein))
             protein.protein = True
+        elif protein is None:
+            raise ValueError('Protein needs to be set for structure based '
+                             'scoring')
         # trigger cache
         protein.atom_dict
 
         if type(function) is str:
```
Comment (marta-sd) [inline oddt/virtualscreening.py]: use `isinstance` instead of `type`

File: tests/test_virtualscreening.py [hunk 2]
```diff
@@ -152,3 +163,83 @@ def test_vs_similarity():
...
+                                               'rfscore_v2', 'rfscore_v3'])
+            data = pd.read_csv(csvfile.name)
+            assert_equal(len(data.columns), 4)
+            assert_in('nnscore', data.columns)
+            assert_in('rfscore_v1', data.columns)
+            assert_in('rfscore_v2', data.columns)
+            assert_in('rfscore_v3', data.columns)
+
+    for f in filenames:
+        os.unlink(f)
```
Comment (marta-sd) [inline tests/test_virtualscreening.py]: What about symlinks created for different PDBbind versions?

Comment (mwojcikowski) [reply tests/test_virtualscreening.py]: Good point

File: oddt/virtualscreening.py (hunk 1 above)
Comment (mwojcikowski) [reply oddt/virtualscreening.py]: Done

**Pull request merged**